test/libsolidity/smtCheckerTests/imports/import_as_module_1.sol

```diff
@@ -0,0 +1,20 @@
+==== Source:  ====
```

Comment:
Might need a source name for the AST import tests.

Reply:
ah ok so that's why

Reply:
Looks like the source name means you need to update the test expectations, since it seems to occur in there now.